wagtail/documents/static_src/wagtaildocs/js/add-multiple.js

```diff
@@ -11,7 +11,9 @@ $(function () {
     add: function (e, data) {
       var $this = $(this);
       var that = $this.data('blueimp-fileupload') || $this.data('fileupload');
-      var li = $($('#upload-list-item').html()).addClass('upload-uploading');
+      var li = $($('#upload-list-item').content.html()).addClass(
```

Comment:
```suggestion
      var li = $($('#upload-list-item').html()).addClass(
```

Maybe this can just be `html` without the `.content` - please check how jQuery handles contents within template tags.